R/call.R

```diff
@@ -425,3 +425,17 @@ call_args_names <- function(call = NULL) {
   call <- call_info(call, NULL)$call
   names2(call_args_lsp(call))
 }
+
+#' Inspect a call.
+#'
+#' This function is useful for quick testing and debugging when you
+#' manipulate expressions and calls. It lets you check that a function
+#' is called with the right arguments. This can be useful in unit
+#' tests for instance.
```

Comment:
Probably should note that this is just a wrapper around `match.call()` that makes the semantics clearer